java/java.project.ui/src/org/netbeans/modules/java/project/ui/NewJavaFileWizardIterator.java

```diff
@@ -241,17 +242,28 @@ private void addRequires(FileObject createdFile, Set<String> requiredModuleNames
             final JavaSource src = JavaSource.forFileObject(createdFile);
             if (src != null) {
                 final Set<String> mNames = requiredModuleNames;
-                src.runModificationTask((WorkingCopy copy) -> {
-                    copy.toPhase(JavaSource.Phase.RESOLVED);
```

Comment:
Surprisingly this Lambda caused the test to fail on CNFE when instantiating the Iterator (test checks if the instanceCreate works), as nbjavac's *Element were not available at test time. Moving to a subclass delayed linkage (the task actually never runs during test)

Reply:
Fine.